Cargo.toml

```diff
@@ -69,6 +69,9 @@ debug = true
 debug-assertions = true
 
 [workspace.dependencies]
-tokio = { version = "=1.20.1", features = ["full", "tracing"] }
 console-subscriber = "=0.1.7"
-nix = "=0.24.1" # DO NOT BUMP UNTIL NIGHTLY IS UPDATED. Otherwise, it brings too new libc.
+nix = "0.26.1"
+serde-wasm-bindgen = "0.4.5"
+wasm-bindgen = { version = "0.2.83", features = ["serde-serialize"] }
+wasm-bindgen-test = { version = "0.3.33" }
+tokio = { version = "=1.20.1", features = ["full", "tracing"] }
```

Comment:
Do we have any policy what dependencies are kept here? Maybe we should put here every dependency (in another PR of course).

Reply:
Not really, the feature was recently introduced. It makes much easier to manage dependencies versions (and prevent having the same dependency in two different versions), so I slowly move dependencies here when I touch them.